apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -9,6 +9,7 @@
             "type"  : "array",
             "items" : {
                 "type"    : "string",
+                "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$", 
```

Comment:
Does this `\\.` (inside `[^` and `]`) mean no `\` and no `.`?

Reply:
So https is not allowed. So I understand https://github.com/aeternity/epoch/pull/431/files#diff-0769afb81ddf2fb23a397550262da25cR138 could be deleted.

Reply:
>Does this \\. (inside [^ and ]) mean no \ and no .?

yes, the host can not start with either `\` nor with a `.`